webapp/package.json

```
@@ -62,7 +62,15 @@
   "jest": {
     "coverageReporters": [
       "text"
-    ]
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 1,
+        "functions": 1,
+        "lines": 1,
+        "statements": 1
+      }
+    }
```

Thresholds of 1 % are quite weak. I would leave it.
```suggestion
    ]
```